src/content/docs/paper/dev/misc/databases.md

```diff
@@ -192,3 +192,41 @@ Using prepared statements in Java with [`PreparedStatement`](jd:java:java.sql:ja
 helps prevent SQL injection. They separate SQL code from user input by using placeholders, reducing the risk of executing unintended SQL commands.
 **Always** use prepared statements to ensure the security and integrity of your data. Read more about SQL injection
 [here](https://www.baeldung.com/sql-injection).
+
+When using `PreparedStatement` the `login` method will become:
+
+```java
+public void login(String username, String password) throws SQLException {
```

Comment:
if we're handing out code examples of stuff this really should follow the safe standard practices, such as using TWR to ensure that we clean up